src/ucs/sys/netlink.c

```diff
@@ -24,6 +24,7 @@
 #include <sys/socket.h>
 #include <unistd.h>
 
+#define UCS_NETLINK_MESSAGE_DONE(nlh) (nlh->nlmsg_type == NLMSG_DONE)
```

Comment:
`_nlh`

Reply:
prepend predicate (functions and variables) with `_is` - in several places here.